ReSharper.FSharp/test/data/features/service/typingAssist/XmlDoc - Indent.fs.gold

```diff
@@ -0,0 +1,8 @@
+﻿// ${CHAR:<}
+module Module
+
+///      <summary>
+///      {caret}
+///      </summary>
+///      <param name="x"></param>
```

Comment:
Is this the same logic as in C#? I'd prefer it not to add these extra spaces to be honest

Reply:
In C# it's always insert a single space before tags

Reply:
Let's have a single space too?